protected/humhub/modules/user/models/Group.php

```diff
@@ -82,6 +84,21 @@ public function validateShowAtRegistration($attribute, $params)
         }
     }
 
+    /**
+     * Validate default group
+     * @param string $attribute
+     */
+    public function validateIsDefaultGroup($attribute)
+    {
+        if ($this->is_admin_group && $this->is_default_group) {
+            $this->addError($attribute, 'Admin group can\'t be a default group!');
```

Comment:
Please use Yii::t
The administrators group cannot be marked as the default group.